[IMP] Helpdesk: closing tickets update (#12551)
odoo/documentation: content/applications/services/helpdesk/advanced/close_tickets.rst

> Closing tickets

Reviewer: Nitpick: Closing or Close?

Manually close solved tickets
=============================

As work on a ticket progresses, it is moved along to the next stage in the pipeline. When the issue
is solved, the ticket is moved to a *folded* stage. This marks the ticket as *closed*.

To fold a stage, navigate to the :menuselection:`Helpdesk` dashboard and click on a team to open the
pipeline. Hover over a stage's heading, and then click the :icon:`fa-gear` :guilabel:`(gear)` icon
that appears in the top-right corner of that stage's Kanban column.

.. image:: close_tickets/closing-edit-stage-gear.png
   :alt: View of stage on Helpdesk pipeline with emphasis on gear icon and edit stage option.

.. warning::
   Clicking the gear icon also displays the option to :guilabel:`Fold` the stage. This setting folds
   the stage *temporarily* to simplify the Kanban view. This does **not** close the tickets in this
   stage. It also does not permanently fold the stage. If a stage needs to be folded so the tickets
   can be marked as closed, continue following the steps below.

> From the menu that appears, select :guilabel:`Edit`. This will open the stage's settings. Tick the

Reviewer: ```suggestion
From the menu that appears, select :guilabel:`Edit`. This opens the stage's settings. Tick the
```

checkbox labeled :guilabel:`Folded in Kanban`, and then :guilabel:`Save & Close` to confirm the

> changes. Now, tickets that reach this stage are considered as *closed*.

Reviewer 2: ```suggestion
changes. Now, tickets get *closed* when they reach this folded stage.
```

   .. image:: close_tickets/closing-folded-setting.png
      :alt: Stage settings page.

Automatically close inactive tickets
====================================

Tickets that are inactive for a set period of time can be automatically closed. At that point, they
are moved to a folded stage.

Go to the team's settings page by going to :menuselection:`Helpdesk --> Configuration --> Helpdesk
Teams`. Under the :guilabel:`Self-Service` section, enable :guilabel:`Automatic Closing`.

> If one of the team's stages is set to be folded in the Kanban view, it is the default selection in

Reviewer 2: ```suggestion
If only one of the team's stages is folded in the Kanban view, it is the default selection in
```

the :guilabel:`Move to Stage` field. If the team has more than one folded stage, the stage that

> occurs first in the pipeline is the default. If no stage is folded, the default selection is the

Reviewer 2: ```suggestion
occurs first in the pipeline is the default for this field. If no stage is folded, the default selection is the
```

   - :guilabel:`Automatic Closing`: *checked*
   - :guilabel:`Move to Stage`: `Solved`

>    - :guilabel:`After` `7` :guilabel:`days of inactivity`

Reviewer: Good rst formatting catch!
